app/views/feedback/feedback.html.haml

```diff
@@ -18,17 +18,31 @@
         %p
           = t('.callout')
           = render partial: 'feedback/claim_edit_alert' if referrer_is_claim?(@feedback.referrer)
+        
+      %p.govuk-body{:class => "govuk-!-margin-bottom-6"}
```

Comment:
Standard: No rocket hashes in views
`:class => "govuk-!-margin-bottom-6"` should be `class: 'govuk-!-margin-bottom-6'`